python_modules/libraries/dagster-dbt/dagster_dbt/asset_defs.py

```diff
@@ -573,7 +573,7 @@ def load_assets_from_dbt_project(
             `dagster_auto_materialize_policy={"type": "lazy"}` will result in that model being assigned
             `AutoMaterializePolicy.lazy()`. Deprecated: instead, configure auto-materialize
             policies on a dbt resource's meta field.
-        node_info_to_definition_metadata_fn (Dict[str, Any] -> Optional[Dict[str, MetadataUserInput]]): [Deprecated]
+        node_info_to_definition_metadata_fn (Dict[str, Any] -> Optional[Dict[str, RawMetadataMapping]]): [Deprecated]
```

Comment:
What should we do about these private type aliases in our API docs? Seems not ideal, but maybe not worth fretting about.

Reply:
It's possible newer versions of sphinx have an option to resolve these during docs generation-- otherwise, we could probably hack it in in `dagster-sphinx`. I'll put it in the backlog.